environment.yml

```diff
@@ -72,6 +72,7 @@ dependencies:
   - matplotlib>=2.2.2  # pandas.plotting, Series.plot, DataFrame.plot
   - numexpr>=2.6.8
   - scipy>=1.1
+  - tabulate
```

Comment:
should this go in the "optional for io" block below?  should any particular minimum version be required?